static/app/components/group/sidebarSectionTitle.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
```

Comment:
Maybe this component should not live inside `app/components/group/` because it's not used only on the issue details page.